.prettierrc.js

```diff
@@ -4,13 +4,14 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     https://www.apache.org/licenses/LICENSE-2.0
+//      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
The lack of an `s` in the `http` is weird.

Reply:
https://github.com/googleapis/gapic-generator-typescript/pull/386

Reply:
Thanks 👍

Reply:
I have updated the docker image, please regen and this one will disappear :)